src/widgets/vitals/vitals-chart.component.tsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import styles from "./vitals-chart.component.scss";
+import "@carbon/charts/styles.css";
+import { RadioButtonGroup, RadioButton } from "carbon-components-react";
+import { PatientVitals } from "./vitals-biometrics.resource";
+import { LineChart } from "@carbon/charts-react";
+import { LineChartOptions } from "@carbon/charts/interfaces/charts";
+import { ScaleTypes } from "@carbon/charts/interfaces/enums";
+import dayjs from "dayjs";
+
+interface vitalsChartData {
+  title: string;
+  value: number | string;
+  groupName:
+    | "Blood Pressure"
+    | "Oxygen Saturation"
+    | "Temperature"
+    | "Respiratory Rate";
+}
+
+interface VitalsChartProps {
+  patientVitals: Array<PatientVitals>;
+  conceptsUnits: Array<string>;
+}
+
+const VitalsChart: React.FC<VitalsChartProps> = ({
+  patientVitals,
+  conceptsUnits
+}) => {
+  const [chartData, setChartData] = React.useState([]);
+  const [
+    bloodPressureUnit,
+    ,
+    temperatureUnit,
+    ,
+    ,
+    ,
+    oxygenSaturationUnit,
+    ,
+    respiratoryRateUnit
+  ] = conceptsUnits;
+  const [selectedVitalSign, setSelecteVitalsSign] = React.useState<
+    vitalsChartData
+  >({
+    title: `BP (${bloodPressureUnit})`,
+    value: "systolic",
+    groupName: "Blood Pressure"
+  });
+
+  React.useEffect(() => {
+    const chartData = patientVitals
+      .map(vitals => {
+        return {
+          group: selectedVitalSign.groupName,
+          key: dayjs(vitals.date).format("DD-MMM"),
+          value: vitals[selectedVitalSign.value]
+        };
+      })
+      .sort((a: any, b: any) => b.date - a.date);
```

Comment:
Seems to me that this `sort` would have no effect, since there is no `date` attribute. Was the sort operation supposed to be done before the mapping?